{"uid":"77ae660bc2d1b6ab","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"d8696551571014a71a874c85cadaf3fe","time":{"start":1750730217275,"stop":1750730217275,"duration":0},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1750730210816,"stop":1750730210816,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"session_request","time":{"start":1750730210816,"stop":1750730210816,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1750730210816,"stop":1750730210816,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1750730210816,"stop":1750730210816,"duration":0},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/conftest.py\", line 60, in data\n    with open(json_path, encoding=\"utf-8\") as json_file:\n         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","steps":[{"name":"public ip address","time":{"start":1750730219664,"stop":1750730219677,"duration":13},"status":"passed","steps":[],"attachments":[{"uid":"f1d39a355f703058","name":"HTTP logs of http://checkip.amazonaws.com/","source":"f1d39a355f703058.txt","type":"text/plain","size":356}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"9b59931bb6cc5bb0","name":"Cookies","source":"9b59931bb6cc5bb0.json","type":"application/json","size":4117},{"uid":"b703c41086b9f62a","name":"Session Storage","source":"b703c41086b9f62a.json","type":"application/json","size":359},{"uid":"5e3b16c257506ec9","name":"Local Storage","source":"5e3b16c257506ec9.json","type":"application/json","size":548},{"uid":"9c78ce4d509a60f","name":"Full Page Screenshot","source":"9c78ce4d509a60f.png","type":"image/png","size":1363660},{"uid":"b86b719a635c4d64","name":"URL","source":"b86b719a635c4d64.uri","type":"text/uri-list","size":23},{"uid":"2d855dd9f963469f","name":"Network Logs","source":"2d855dd9f963469f.json","type":"application/json","size":49304}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":7,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"session_request::0","time":{"start":1750730277279,"stop":1750730277279,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"severity","value":"blocker"},{"name":"epic","value":"Security"},{"name":"feature","value":"Login"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az810-840"},{"name":"thread","value":"2059-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'nirt236@gmail.com'"},{"name":"password","value":"'123456'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":10,"skipped":0,"passed":0,"unknown":0,"total":10},"items":[{"uid":"8945e9bc7a4e6895","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/693//#testresult/8945e9bc7a4e6895","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1750644190382,"stop":1750644190382,"duration":0}},{"uid":"1326326508233adc","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/692//#testresult/1326326508233adc","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1750557882942,"stop":1750557882942,"duration":0}},{"uid":"b82b360314a96387","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/691//#testresult/b82b360314a96387","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1750470803730,"stop":1750470803730,"duration":0}},{"uid":"4fe4134c9dd818a6","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/690//#testresult/4fe4134c9dd818a6","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1750384537948,"stop":1750384537948,"duration":0}},{"uid":"2b78500c439473cb","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/689//#testresult/2b78500c439473cb","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1750298179251,"stop":1750298179251,"duration":0}},{"uid":"8f7f0fa350c04f86","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/688//#testresult/8f7f0fa350c04f86","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1750211730436,"stop":1750211730436,"duration":0}},{"uid":"7920df625fdb639b","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/687//#testresult/7920df625fdb639b","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1750125377827,"stop":1750125377827,"duration":0}},{"uid":"347f8c4f817facaa","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/686//#testresult/347f8c4f817facaa","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1750039172870,"stop":1750039172870,"duration":0}},{"uid":"1a5f90ac333e1d6f","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/685//#testresult/1a5f90ac333e1d6f","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1749953132005,"stop":1749953132005,"duration":0}}]},"tags":["run(order=3)","security"]},"source":"77ae660bc2d1b6ab.json","parameterValues":["'nirt236@gmail.com'","'123456'"]}